### ISSUE TITLE
Re-writing history

### ISSUE THREAD

**Issue by stephenrwalli** [sfosc/sfosc#58]
In the book introduction, it says, “Back then, it was a cutting-edge idea - participating in open source was a radical act.“ I beg to differ. We had been collaborating on software using liberal licenses since 1980. There had been almost 20 years of experiments from MIT, UC Berkeley, Stallman’s GPL, Perl and the Artistic License, the Apache project, and the beginnings of corporate licenses with the first Mozilla license. We’d been collaborating on software without licenses being forced by copyright law in 1980 for almost 30 years before then.

**Comment by Beanow** [comment]
Welcome and thanks for pointing this out :+1:
It seems like a good idea to get the facts straight here. Would you happen to have a good source for background reading on this to cite from? Or can you suggest an alternative phrasing?

**Comment by nothingismagick** [comment]
Perhaps @stephenrwalli could contribute a "witness" citation... Would that be a good idea and if so would you be open to this?

**Comment by vsoch** [comment]
If a "witness" citation is added, should others be pursued to contribute experiences? It would be good to extract a lot of the history that still lives in people's heads :)

**Comment by Beanow** [comment]
Like the idea of witness citation. As far as getting our facts straight on the pages here I'm happy with either external resources or such a citation. If there are multiple people willing to contribute citations, certainly wouldn't turn that down either :smile:

But, since SFOSC is opinionated and [makes no attempt](https://github.com/sfosc/sfosc/pull/42#issuecomment-491349890) at being neutral, I think pursuing to archive these experiences _here_ doesn't do the information justice. In that case I think we should try to find a better home for that kind of wealth. Maybe a separate thing, maybe contribute to wikipedia, I'm not sure.

**Comment by nothingismagick** [comment]
I just wanted to drop this here, because if we're considering a historical bit, this seems highly relevant:

http://www.catb.org/~esr/writings/cathedral-bazaar/cathedral-bazaar/index.html

**Comment by pradyunsg** [comment]
Currently, the relevant paragraph is:

> Since the term “Open Source” was first adopted in 1997, it has become the dominant software development model around the world. Back then, it was a cutting-edge idea - participating in open source was a radical act. Simply participating in it required a philosophical background. Today, you can actively participate in Open Source without any background at all in its philosophy. Open Source is in the water for the new generation of software developers.

Here's a proposal for replacing it with something else, a trip down history lane:

> The term “Open Source” was first adopted in 1997. At the time, it was common practice for software to be distributed in binary/compiled form with restrictive licensing and no access to the source code. Meanwhile, for over a decade, liberal licenses had been used for software developed at universities for fostering collaboration. The underlying philosophy behind these licenses being, to foster collaborative communities where a single institution can not limit or imposes restrictions on the collaboration. The late 1990s and early 2000s saw a significant increase in the adoption of this philosophy for software development, and a corresponding increase in licensing of software under such liberal licenses.
>
> Since then, it has become the dominant software development model around the world. Today, you can actively participate in Open Source without any background at all in its philosophy. Open Source is in the water for the new generation of software developers.

Thoughts are welcome. Lemme know if this looks like an improvement to y'all and I'll be happy to file a PR to iterate on this (and improve the language/phrasing etc), if we're OK with the general approach here. :)

---

A lot of this is based on multiple readings of https://en.wikipedia.org/wiki/History_of_free_and_open-source_software, https://en.wikipedia.org/wiki/Free_software_movement and a bunch more links that I'm not in the mood to dig out of my history. I don't know how important we're considering citations here so lemme know if that's super needed.

---

I'll note that I've noticed that there are some speculative blog posts/notes on Wikipedia regarding when "Open Source" was coined/adopted. However, I think that's a little OT for this issue. AFAICT based on some light reading, 1997 or 1998 were the relevant year so we might want to change that year up to 1997/98.

**Comment by Beanow** [comment]
Thank you @pradyunsg! Would like to ask @stephenrwalli first of all what he thinks of the proposal.
Also @adamhjk who wrote the original.

Otherwise I think you've already done more research than I have and will defer to your judgement there. :+1:

**Comment by vsoch** [comment]
I like it a lot! A few suggestions from me:

> The underlying philosophy behind these licenses being, to foster collaborative communities where a single institution can not limit or imposes restrictions on the collaboration.

I would change "being" to just "was." If someone were speaking this, "being" would sound more eloquent, but reading it, I think just being clear and direct with "was" sounds better.

> Since then, it has become the dominant software development model around the world. 

Since you are opening a new paragraph, we should refer again to "open source" instead of "it."

The last sentence about water is sweet, but I think it's a bit too metaphorical for the general feeling of the other content on the site.

**Comment by nothingismagick** [comment]
Yeah, I think you should go ahead and make a PR. While I agree with @Beanow about being inclusive of the original issue reporter @stephenrwalli and @adamhjk - this issue has been open since May.

**Comment by nothingismagick** [comment]
@vsoch - I agree with the water metaphor too, but its from the original text by Adam - so that should be a different discussion point.

**Comment by pradyunsg** [comment]
Filed #134, which also incorporates inputs from @vsoch.

**Comment by adamhjk** [comment]
@stephenrwalli as someone who also existed in 1997, it was pretty radical, if you weren't present at the creation. Even if you were, timeline wise - I worked with plenty of folks in 1997 who started their careers in the late 70s and early 80s, and open source and free software were radical to them. 

I wasn't claiming (and don't) that it was invented in 1997, or that the history of it began then (it didn't.) Simply that, in 1997, the barrier to entry was significantly higher, and it was more radical - you were unlikely to trip your way into the free and open source community as it was in 1997, whereas that's incredibly likely now.

@pradyunsg put in a PR for this that I closed, but I think its worth adding a history section to the site.

**Comment by vsoch** [comment]
I wish I could offer more comment, but I was 10 and playing Quake Team Fortress and Duke Nukem 3D. I feel like I missed out on a lot - I wonder if somewhere we could better tell stories about these communities, and what it was like. How were they found then? What did open source mean then that is now different? It would be really wonderful to be able to put someone (like myself) in the shoes of someone that experienced the culture at that time. Otherwise, I (and future folks) will just never know :/

**Comment by adamhjk** [comment]
@vsoch It does take quite a bit of hunting to unearth the archeology of it. I think a big part of it was that everything was smaller then, which meant that (while still being massive) things still often "felt" of a piece. 

A good book for what it felt like in 1993 is Howard Rheingold's ["The Virtual Community"](https://www.rheingold.com/vc/book/intro.html)
